packages/terra-navigation-side-menu/src/NavigationSideMenu.jsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'terra-base/lib/baseStyles';
+import List from 'terra-list';
+import ActionHeader from 'terra-clinical-action-header';
+import ContentContainer from 'terra-content-container';
+import MenuItem from './_MenuItem';
+
+const KEYCODES = {
+  ENTER: 13,
```

Comment:
Do we have (or want to have) a more common place for common key codes?  I can't remember if this has been brought up before.

Reply:
Yeah - https://github.com/cerner/terra-core/issues/1010. We should probably discuss escalating this issue, we have key codes scattered throughout a lot of components.